web3/providers/eth_tester/middleware.py

```diff
@@ -64,7 +64,58 @@ def is_hexstr(value: Any) -> bool:
 is_not_named_block = complement(is_named_block)
 
 
-TRANSACTION_KEY_MAPPINGS = {
+# --- Request Mapping --- #
```

Comment:
I just re-organized / re-named a few of these so that it is more clear which are request formatters and which are result formatters.

Reply:
Yeah, that's much better, thanks!